### DISCUSSION TITLE
use failed

### DISCUSSION THREAD

**Discussion by JXAUDZF** [ConesaLab/SQANTI3#264] [General] [answered]
Unable to import err_correct_w_genome or sam_to_gff3.py! Please make sure cDNA_Cupcake/sequence/ is in $PYTHONPATH.

**Comment by carolinamonzo** [comment, marked as answer]
Hi @JXAUDZF the error is self explanatory, you have to put the path to cDNA_Cupcake in your pythonpath. This is explained in the installation instructions in sqanti.
